{"name": "test_login_params[data_for_login2]", "status": "passed", "attachments": [{"name": "log", "source": "f3e7e984-c09f-4556-8c3c-5e0b7931ea62-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh@gmail.com', '48465', 'Fail')"}], "start": 1746304506111, "stop": 1746304508906, "uuid": "76b070ae-f371-403f-8f34-7406c32e4de4", "historyId": "cd1ac65cf3cc05caccdeb520d07aec8a", "testCaseId": "47f970e602e4eefb1a3e5455dd555a36", "fullName": "testCases.test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "23568-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_login_params"}]}